gsplat/strategy/ops.py

```diff
@@ -68,19 +68,22 @@ def _update_param_with_optimizer(
         names = list(params.keys())
 
     for name in names:
+        param = params[name]
+        new_param = param_fn(name, param, param.requires_grad)
+        params[name] = new_param
+        if name not in optimizers:
+            assert not param.requires_grad
```

Comment:
Add a msg to explain. sth like `assert not param.requires_grad, f"param {name} does not present in the optimizer, it's requires_grad should be False, but found True"`